src/it/scala/com/wavesplatform/it/Docker.scala

```diff
@@ -183,12 +248,18 @@ class Docker(suiteConfig: Config = ConfigFactory.empty,
       nodes.asScala.foreach { node =>
         node.close()
         client.stopContainer(node.nodeInfo.containerId, 0)
-        client.disconnectFromNetwork(node.nodeInfo.containerId, wavesNetworkId)
+        disconnectFromNetwork(node)
         saveLog(node)
         client.removeContainer(node.nodeInfo.containerId, RemoveContainerParam.forceKill())
       }
 
-      client.removeNetwork(wavesNetworkId)
+      Try(client.removeNetwork(wavesNetwork.id)) match {
```

Comment:
Is `Try(...) match {` any better than plain old `try { ... } catch { ... }` in this case?

Reply:
Nope, I'll fix it